tests/buildkite/pipeline-nightly.yml

```diff
@@ -0,0 +1,37 @@
+# Nightly CI pipeline, to test against dev versions of dependencies
+
+env:
+  DOCKER_CACHE_ECR_ID: "492475357299"
+  DOCKER_CACHE_ECR_REGION: "us-west-2"
+  DISABLE_RELEASE: "1"
+    # Skip uploading artifacts to S3 bucket
+    # Also, don't build all CUDA archs; just build sm_75
+  USE_DEPS_DEV_VER: "1"
+    # Use dev versions of RAPIDS and other dependencies
+steps:
+  #### -------- CONTAINER BUILD --------
+  - label: ":docker: Build containers"
+    commands:
+      - "tests/buildkite/build-containers.sh gpu_build_centos7"
```

Comment:
RAPIDS no longer supports CentOS 7. If this command requires RAPIDS, it will need to use something with glibc >=2.28 (Rocky Linux 8 has 2.28, Ubuntu 20.04 has 2.31).

Reply:
This container is solely used for building XGBoost for manylinux2014 standard. Should we migrate XGBoost to glibc >= 2.28 too?

Reply:
Opened https://github.com/dmlc/xgboost/issues/10357. Since CentOS 7 will reach EOL soon, we should move to rockylinux8.